pkg/apis/core/types.go

```diff
@@ -918,6 +918,11 @@ type QuobyteVolumeSource struct {
 	// Default is no group
 	// +optional
 	Group string
+
+	// Tenant owning the given Quobyte volume in the Backend
+	// Used with dynamically provisioned Quobyte volumes, value is set by the plugin
+	// +optional
+	Tenant string
```

Comment:
Does the `tenant` have a specified format? Any character can be used? Does it have any size limit? There should be some validation in [validation.go](https://github.com/kubernetes/kubernetes/blob/86165ac87889735abec1c482fb869a7cd10303a3/pkg/apis/core/validation/validation.go#L872)

I don't want users putting megabytes of data here.